cmd/apply.go

```diff
@@ -32,6 +32,11 @@ var applyCommand = &cli.Command{
 			Usage:     "Path to cluster backup archive to restore the state from",
 			TakesFile: true,
 		},
+		&cli.StringFlag{
+			Name:  "check-semver",
```

Comment:
i would make this a boolflag (which defaults to false) and call it `--no-check-semver` or `--disable-version-check` or something like that.

Reply:
renamed it to `disable-downgrade-check` and made it boolflag